### DISCUSSION TITLE
iPad not playing timelapse

### DISCUSSION THREAD

**Discussion by cleggers21** [aaronwmorris/indi-allsky#1475] [Q&A] [answered]
Anybody know why my IPad will not play Timelapse videos. 
I’ve tried 3 different browsers.
They play ok on my PC and Android phone.
I’m using a RPi5 8gb.

**Comment by aaronwmorris** [comment]
On the Image config tab, try adding `-level 3.1` to "FFMPEG Extra Options".  This supposedly increases compatibility with Apple devices.

**Reply by cleggers21** [reply]
Thanks Aaron, tried that, rebooted but still no streaming. 
They do run if I  play them through VNC.

**Reply by aaronwmorris** [reply]
You will need to regenerate (delete and recreate) the timelapse to have the feature applied to the video.

**Reply by cleggers21** [reply]
Tried on my Mac and they stream ok so I guess it’s just my very old iPad although it’s up to date.

**Reply by cleggers21** [reply]
Ok, I’ll recreate and try.

**Reply by aaronwmorris** [reply, marked as answer]
Its mainly IOS devices that have a problem.  There was an extensive discussion about this on the Allsky github.

https://github.com/AllskyTeam/allsky/issues/3286

**Comment by cleggers21** [comment]
Hi Aaron
Still no streaming on my iPad (v5 2018).
I've tried both  -level 3.1 and -level 3.1 -refs 11 both with reboots and generating new timelapse.
My workaround is to either use my PC or Android or access the TL's via VNC.

**Reply by aaronwmorris** [reply]
I wish I had more experience with media generation with FFMPEG.  The amount of options available is mind boggling and much of the information available [for ffmpeg] on the Internet is wrong or based on guesses.
